templates/conf/fluent.conf.erb

```diff
@@ -164,6 +164,7 @@
    include_tag_key true
    host "#{ENV['FLUENT_GRAYLOG_HOST']}"
    port "#{ENV['FLUENT_GRAYLOG_PORT']}"
+   protocol "#{ENV['FLUENT_GRAYLOG_PROTOCOL']} || 'udp'"
```

Comment:
Does this work on your environment? After evaluated this line, the parameter become `"tcp || udp"`, not `"tcp"`